examples/sequence_tagging/conll_reader.py

```diff
@@ -0,0 +1,266 @@
+                    word_count[word] += min_occur + 1
+                elif file_path == train_path:
+                    word_count[word] += 1
+
+                nid = ner_vocab[ner]
+
+    print("Total Vocabulary Size: %d" % len(word_count))
+    for word in word_count:
+        if word_count[word] > min_occur:
+            wid = word_vocab[word]
+
+    print("Word Vocabulary Size: %d" % len(word_vocab))
+    print("Character Alphabet Size: %d" % len(char_vocab))
+    print("NER Alphabet Size: %d" % len(ner_vocab))
+
+    word_vocab = defaultdict(lambda: UNK_WORD, word_vocab)
+    char_vocab = defaultdict(lambda: UNK_CHAR, char_vocab)
+    ner_vocab = defaultdict(lambda: UNK_NER, ner_vocab)
+
+    i2w = {v: k for k, v in word_vocab.items()}
+    i2n = {v: k for k, v in ner_vocab.items()}
+    return (word_vocab, char_vocab, ner_vocab), (i2w, i2n)
+
+
+def read_data(source_path, word_vocab, char_vocab, ner_vocab,
+              normalize_digits=True):
+    data = []
+    print('Reading data from %s' % source_path)
+    counter = 0
+    reader = CoNLLReader(source_path, word_vocab, char_vocab, ner_vocab)
+    inst = reader.getNext(normalize_digits)
+    while inst is not None:
+        counter += 1
+        sent = inst.sentence
+        data.append([sent.word_ids, sent.char_id_seqs, inst.ner_ids])
+        inst = reader.getNext(normalize_digits)
+
+    reader.close()
+    print("Total number of data: %d" % counter)
+    return data
+
+
+def iterate_batch(data, batch_size, shuffle=False):
+    if shuffle:
+        np.random.shuffle(data)
+
+    for start_idx in range(0, len(data), batch_size):
+        excerpt = slice(start_idx, start_idx + batch_size)
+        batch = data[excerpt]
+
+        batch_length = max([len(batch[i][0]) for i in range(len(batch))])
+
+        wid_inputs = np.empty([len(batch), batch_length], dtype=np.int64)
+        cid_inputs = np.empty([len(batch), batch_length, MAX_CHAR_LENGTH],
+                              dtype=np.int64)
+        nid_inputs = np.empty([len(batch), batch_length], dtype=np.int64)
+        masks = np.zeros([len(batch), batch_length], dtype=np.float32)
+        lengths = np.empty(len(batch), dtype=np.int64)
+
+        for i, inst in enumerate(batch):
+            wids, cid_seqs, nids = inst
+
+            inst_size = len(wids)
+            lengths[i] = inst_size
+            # word ids
+            wid_inputs[i, :inst_size] = wids
+            wid_inputs[i, inst_size:] = PAD_WORD
+            for c, cids in enumerate(cid_seqs):
+                cid_inputs[i, c, :len(cids)] = cids
+                cid_inputs[i, c, len(cids):] = PAD_CHAR
+            cid_inputs[i, inst_size:, :] = PAD_CHAR
+            nid_inputs[i, :inst_size] = nids
+            nid_inputs[i, inst_size:] = PAD_NER
+            masks[i, :inst_size] = 1.0
+
+        yield wid_inputs, cid_inputs, nid_inputs, masks, lengths
+
+
+def load_glove(filename, emb_dim, normalize_digits=True):
```

Comment:
I thought we had a `load_glove` method inside `tx.data`? What are the differences here?